app/build.gradle.kts

```diff
@@ -145,31 +157,45 @@ dependencies {
     androidTestImplementation(Dependencies.Health.androidXEspresso)
 }
 
-tasks.withType<Test> {
-    useJUnitPlatform()
-    testLogging {
-        outputs.upToDateWhen { false }
-        showStandardStreams = true
+tasks {
+    withType<Detekt> {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
I don't think this is needed?

Reply:
Detekt recommends it in the setup. Idk.

Reply:
I didn't need it for other projects I've used Detekt and it works fine for those

Reply:
Afaik it serves as a hint to detekt. So not 'needed', but might be helpful?

Reply:
https://detekt.github.io/detekt/gradle.html#using-type-resolution

The default `detekt` task already uses Java 1.8

Reply:
I can't read that from there, but I'll remove it ig.